Frameworks/Foundation/NSURLSessionConfiguration.mm

```diff
@@ -71,7 +71,7 @@ - (instancetype)initBackgroundSession:(NSString*)identifier {
 */
 - (id)copyWithZone:(NSZone*)zone {
     NSURLSessionConfiguration* copy = [[[self class] allocWithZone:zone] init];
-    copy.identifier = _identifier;
+    copy.identifier = [_identifier copy];
```

Comment:
Bad! This is going through the property setter, and the property is declared as taking a copy already.
This will 1) make an unnecessary copy and 2) leak it.